kubernetes/linux/setup.sh

```diff
@@ -73,7 +73,7 @@ docker_cimprov_version=$(sudo tdnf list installed | grep docker-cimprov | awk '{
 echo "DOCKER_CIMPROV_VERSION=$docker_cimprov_version" >> packages_version.txt
 
 #install fluent-bit
-sudo tdnf install fluent-bit-3.1.9 -y
+sudo tdnf install azcu-fluent-bit-4.0.9 -y
 echo "$(fluent-bit --version)" >> packages_version.txt
```

Comment:
1. since this major version upgrade, did you verify if any the existing config properties are deprecated? 2. since this major version upgrade, we would need to validate all the scenarios. 3. we would need also validate the resource usage (CPU and memory) with and without this version update.

Reply:
attached resource usage in describe